projects/plugins/jetpack/extensions/blocks/recurring-payments/button/util.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import formatCurrency from '@automattic/format-currency';
+
```

Comment:
I think we can delete this file, these functions are not being used in our code base.

Reply:
Thanks, looks like this was already out of use